build.gradle

```diff
@@ -782,6 +783,7 @@ project(':connect:runtime') {
     compile "org.eclipse.jetty:jetty-servlet:$jetty_version"
     compile "com.fasterxml.jackson.jaxrs:jackson-jaxrs-json-provider:$jackson_version"
     compile "org.glassfish.jersey.containers:jersey-container-servlet:$jersey_version"
+    compile "org.reflections:reflections:$reflections_version"
```

Comment:
I know we discussed this briefly offline, but worth double checking and raising the point with others who might have an opinion. Are we fine with adding this dependency, which also pulls in some other libraries? Looks like at least 3 new jars and a bump from 8.5MB -> 11MB of jars in `connect:runtime` (mostly due to Guava).

I think the most notable addition is probably Guava. This is very widely used, so I don't want to end up with any conflicts with connector plugins, but I guess Guava is also probably extremely careful about compatibility? @junrao might want to chime in since I know he wasn't thrilled about the number of dependencies pulled in to support the REST API (which admittedly was a much larger number than these).